### ISSUE TITLE
bug(mobile): image editor icon under navigation bar

### ISSUE THREAD

**Issue by waclaw66** [immich-app/immich#11956]
### The bug

@Yuvi-raj-P 

![Screenshot_20240821_151539](https://github.com/user-attachments/assets/be946790-fa1c-4c4f-a4a2-5fd387da8f19)

Additionally there is completely missing translation of the editor.

### The OS that Immich Server is running on

Fedora 40

### Version of Immich Server

v1.112.1

### Version of Immich Mobile App

v1.112.1

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
not relevant
```

### Your .env content

```Shell
not relevant
```

### Reproduction steps

```bash
1. view asset
2. click Edit button
3. see Crop button under navigation bar
```

**Comment by Yuvi-raj-P** [comment]
Hello, Thanks for reporting the issue. This was recently fixed, coincidentally on the latest pull request: https://github.com/immich-app/immich/pull/11929 . However could you tell me the device that you're using and its OS just for some more information.

**Comment by waclaw66** [comment]
> However could you tell me the device that you're using and its OS just for some more information.
Samsung Galaxy S20 FE, Android 13

**Comment by Yuvi-raj-P** [comment]
Fixed! Pull request: #12018 you could close now or let me know if it works
